containers/withUserContainer.ts

```diff
@@ -0,0 +1,15 @@
+import { Context } from '../@types/helpers'
+import _ from 'lodash'
+
+export const withUserContainer =
+  <Type, ArgsType>(
+    resolver: (_parent: void, args: ArgsType, ctx: Context) => Type
+  ) =>
+  async (_parent: void, args: ArgsType, ctx: Context) => {
+    const { req } = ctx
+
+    const authorId = _.get(req, 'user.id', undefined)
```

Comment:
undefined is unnecessary because lodash get will automatically return undefined when the looked up property(user.id) is not found in the object(req)